r/vignettes/arrow.Rmd

```diff
@@ -150,42 +159,52 @@ In the tables, entries with a `-` are not currently implemented.
 | float32           | double                       |
 | float64           | double                       |
 | utf8              | character                    |
-| binary            | arrow_binary ^5^             |
-| fixed_size_binary | arrow_fixed_size_binary ^5^  |
+| large_utf8        | character                    |
+| binary            | arrow_binary ^3^             |
+| large_binary      | arrow_large_binary ^3^       |
+| fixed_size_binary | arrow_fixed_size_binary ^3^  |
 | date32            | Date                         |
 | date64            | POSIXct                      |
 | time32            | hms::difftime                |
 | time64            | hms::difftime                |
 | timestamp         | POSIXct                      |
-| duration          | -                            |
+| duration          | -^2^                         |
 | decimal           | double                       |
 | dictionary        | factor^4^                    |
-| list              | arrow_list ^6^               |
-| fixed_size_list   | arrow_fixed_size_list ^6^    |
+| list              | arrow_list ^5^               |
+| large_list        | arrow_large_list ^5^         |
+| fixed_size_list   | arrow_fixed_size_list ^5^    |
 | struct            | data.frame                   |
 | null              | vctrs::vctrs_unspecified     |
-| map               | -                            |
-| union             | -                            |
-| large_utf8        | character                    |
-| large_binary      | arrow_large_binary ^5^       |
-| large_list        | arrow_large_list ^6^         |
+| map               | -^2^                         |
+| union             | -^2^                         |
+
+^1^: These integer types may contain values that exceed the range of R's 
+`integer` type (32-bit signed integer). When they do, `uint32` and `uint64` are 
+converted to `double` ("numeric") and `int64` is converted to 
+`bit64::integer64`. This conversion can be disabled (so that `int64` always
+yields a `bit64::integer64` vector) by setting `options(arrow.int64_downcast = FALSE)`.
+
+^2^: Some Arrow data types do not have an R equivalent and will raise an error
+if cast to or mapped to via a schema.
 
-^3^: These integer types may contain values that exceed the range of R's `integer` type (32-bit signed integer). When they do, `uint32` and `uint64` are converted to `double` ("numeric") and `int64` is converted to `bit64::integer64`. This conversion can be disabled (so that `int64` always yields a `bit64::integer64` vector) by setting `options(arrow.int64_downcast = FALSE)`.
+^3^: `arrow*_binary` classes are implemented as lists of raw vectors. 
 
-^4^: Due to the limitation of R `factor`s, Arrow `dictionary` values are coerced to string when translated to R if they are not already strings.
+^4^: Due to the limitation of R factors, Arrow `dictionary` values are coerced
+to string when translated to R if they are not already strings.
```

Comment:
This isn't relevant in this doc, but now I'm curious: what is the limitation of R factors here?

Reply:
Not a clue tbh; I think @ianmcook may(?) have written this text originally, so might have a better idea than me